src/components/AccountAvatars.tsx

```diff
@@ -29,7 +33,7 @@ export const AccountAvatar: React.FC<{
   );
 };
 
-export const AvatarList: React.FC = ({ children }) => {
+export const AvatarList = ({ children }: { children: React.ReactNode }) => {
```

Comment:
Does it make sense to make a `ChildProps` component thats type `{children}` ? i see it being used else where

Reply:
Do you mean a ChildProps component, or a ChildProps type?

Reply:
type sorry